examples/TimoshenkoBeamCase/timoshenko.py

```diff
@@ -85,9 +85,34 @@ class TimoshenkoBeamSimulator(BaseSystemCollection, Constraints, Forcing):
         OneEndFixedBC, constrained_position_idx=(0,), constrained_director_idx=(0,)
     )
     timoshenko_sim.add_forcing_to(unshearable_rod).using(
-        EndpointForces, 0.0 * end_force, end_force, ramp_up_time=final_time / 2.0
+        EndpointForces, 0.0 * end_force, end_force, ramp_up_time=100.0# final_time / 2.0
     )
 
+# Add call backs
+class VelocityCallBack(CallBackBaseClass):
+    """
+    Call back function for continuum snake
+    """
```

Comment:
Can you correct this docstring?

Reply:
My bad. Updated in [dfa9869](https://github.com/GazzolaLab/PyElastica/pull/97/commits/dfa986958533d4919c4436f30dde3327ab629a00).